client/my-sites/people/invite-people/index.jsx

```diff
@@ -361,7 +359,8 @@ const InvitePeople = React.createClass( {
 									disabled={ this.state.sendingInvites } />
 								<FormSettingExplanation>
 									{ this.translate(
-										'(Optional) You can enter a custom message of up to 500 characters that will be included in the invitation to the user(s).'
+										'(Optional) You can enter a custom message of up to 500 characters ' +
```

Comment:
I'd still investigate further that idea ;)
@akirk would that be possible to use multiline translation with template string literal as proposed by @tyxla?